src/db/clients/postgresql.js

```diff
@@ -157,6 +158,22 @@ export function listTableTriggers(client, table) {
   });
 }
 
+export function listSchemas(client) {
+  return new Promise((resolve, reject) => {
+    const sql = `
+    SELECT schema_name
```

Comment:
indention might be like the others https://github.com/eMerzh/sqlectron-core/blob/a88ac7b6981150fa5e67a7e414c13e5250b5185e/src/db/clients/postgresql.js#L102